### PR TITLE
fix(attentive_tag): ecom events user object fix

### PR DESCRIPTION
**Pull request by ujjwal-ab**
## Description of the change

Ecom events user object fix in the output. `externalIdentifiers` object is expected at inside user object, and were sent at root level.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: test/__tests__/data/attentive_tag_output.json [hunk 1]
```diff
@@ -254,8 +254,45 @@
     "files": {}
   },
   {
-    "statusCode": 400,
-    "error": "[Attentive Tag] :: Either email or phone is required"
```
Comment (yashasvibajpai) [inline test/__tests__/data/attentive_tag_output.json:258]: Earlier this was a 400 error but now we are getting a response. Is this expected?

Comment (ujjwal-ab) [reply test/__tests__/data/attentive_tag_output.json:258]: Yes

Comment (yashasvibajpai) [reply test/__tests__/data/attentive_tag_output.json:258]: So, we have external identifiers also apart from `email` and `mobile`. Is this mentioned in the doc or needs any change?

Comment (ujjwal-ab) [reply test/__tests__/data/attentive_tag_output.json:258]: This wasn't mentioned in the doc. But, we can enhance the doc a bit.

**Review by ItsSudip — APPROVED** [review]

**Pull request merged**